.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+run-name: release new version by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: on
+  PIP_NO_CLEAN: on
+  PIP_NO_INPUT: on
+  PIP_PREFER_BINARY: on
+
+permissions:
+  contents: write
+  packages: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  containerize:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Install requirements
+        run: make setup
```

Comment:
Are we doing a local (actions container) install of the project for linting purposes? Do we need to separately lint our manually triggered container builds?

Reply:
this is installing only the requirements.txt, which just contains the global requirements (cruft, pipenv, pre-commit). we use this in our other repos, to pin the versions of project tools, package managers, etc.

i can also simplify it here and just run `pip install pipenv`